doc/lightning-openchannel_bump.7.md

```diff
@@ -0,0 +1,74 @@
+lightning-openchannel\_bump -- Command to initiate a channel RBF
+=====================================================================
+
+SYNOPSIS
+--------
+
+**openchannel_bump** *channel_id* *amount* *initalpsbt*
+
+DESCRIPTION
+-----------
+
+`openchannel_bump` is a RPC command which initiates a channel
+RBF (Replace-By-Fee) for the specified channel. It uses the openchannel protocol
+which allows for interactive transaction construction.
+
+*id* is the id of the channel to RBF.
+
+*amount* is the satoshi value that we will contribute to the channel.
+This value will be _added_ to the provided PSBT in the output which is
+encumbered by the 2-of-2 script for this channel.
```

Comment:
How could the caller set this value?

Why would it use this value?  What if it wants to subtract the fee increase from the channel amount?